feature-about/src/main/java/io/github/droidkaigi/confsched2022/feature/about/About.kt

```diff
@@ -183,10 +201,16 @@ private fun navigateToCustomTab(url: String, context: Context) {
     }
 }
 
+private fun versionName(context: Context) = context.packageManager
+    .getPackageInfo(
+        context.packageName,
+        0
+    ).versionName
```

Comment:
As I implemented it, I was a little suspicious that this code is right. I wanted to use ``BuildConfig.VERSION_NAME``, but I couldn't created BuildConfig even if versionCode is written on ``Project.setupAndroid()``. 

The function is working properly, but is this implementation as intended?